render-test/runner.cpp

```diff
@@ -294,6 +295,31 @@ bool TestRunner::checkRenderTestResults(mbgl::PremultipliedImage&& actualImage,
         }
     }
 
+    // Check network metrics.
+    for (const auto& expected : metadata.expectedMetrics.network) {
+        auto actual = metadata.metrics.network.find(expected.first);
+        if (actual == metadata.metrics.network.end()) {
+            metadata.errorMessage = "Failed to find network probe: " + expected.first;
+            return false;
+        }
+        if (actual->second.requests > expected.second.requests) {
```

Comment:
`!=`? Making less requests should also be an error. For instance a regression could make us request less fonts.

Reply:
Done